### PR TITLE
Exclude smoke-k8s and smoke -docker-json from coverage report.

### PR DESCRIPTION
**Pull request by ArthurKamalov**
* Exclude smoke-k8s and smoke -docker-json from coverage report.
* Add smoke-standalone-27 to coverage report.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -59,10 +59,19 @@ jobs:
           docker_layer_caching: true
       - run:
           command: |
+            mkdir -p coverage-standalone-27 && \
             docker container create --name dummy -v shared_vol:/app alpine && \
```
Comment (ArthurKamalov) [inline .circleci/config.yml]: @czerwingithub, I tried to run a job without this `dummy` container lines, and nothing changed.  Maybe we can remove those lines?

Comment (czerwingithub) [reply .circleci/config.yml]: Eh, let's leave them for now.... I don't fully understand these tests right now.  Just add a comment saying you believe they can be removed.

File: .circleci/config.yml [hunk 2]
```diff
@@ -342,6 +350,5 @@ workflows:
       - coverage:
           requires:
             - unittest-27
-            - smoke-docker-json
             - smoke-docker-syslog
-            - smoke-k8s
```
Comment (czerwingithub) [inline .circleci/config.yml]: I would just comment out the lines for `smoke-docker-json` and `smoke-k8s` with a TODO to re-enable them when they are passing again.

**Comment by czerwingithub** [comment]
@ArthurKamalov We don't need this anymore, so I'm going to close it.  Thanks for putting it forward though.

**Pull request closed without merging**